### ISSUE TITLE
Voting Listings

### ISSUE THREAD

**Issue by TheMan6900** [EmeraldLoc/tag#406]
Basically there would be a little box under level pictures (or not) displaying the names of who voted on said level.

**Comment by Bear64DX** [comment]
I feel like char icons would work better. They're smaller than names so imo it'd be better to have icons

**Comment by TheMan6900** [comment]
how would character select work than?

**Comment by EmeraldLoc** [comment]
I'm sure there's an api for that, haven't looked into it

**Comment by Bear64DX** [comment]
Yeah, plus I never really see char select used for tag. I mainly see it on like mh or smth

**Comment by EmeraldLoc** [comment]
> plus I never really see char select used for tag.

That's pretty much for any mod. Since character select was built for DX, and tag is mainly hosted on coop lobbies, it simply isn't hosted as much.

**Comment by TheMan6900** [comment]
still would be nice if both that and dynamic icons were used for tag.

**Comment by TheMan6900** [comment]
> Yeah, plus I never really see char select used for tag. I mainly see it on like mh or smth

never see mariohunt servers with character select bup

**Comment by Bear64DX** [comment]
I do all the time, especially the ones with like Boykisser (yes its an actual character), and Many Coop Skins. You probably just don't see them cause you host servers a lot

**Comment by Bear64DX** [comment]
I'm intrigued when you say it's built for DX. I know the game says it's meant for DX, but what even makes it for dx?

**Comment by EmeraldLoc** [comment]
I don't quite remember. I believe there are some features for models that coopdx supports but not coop, so not necessarily character select, but the mods that hook into character select. The only functionality that's removed in the cs mod if you aren't using dx is the custom camera status.

**Comment by Bear64DX** [comment]
The only DX exclusive thing I've seen for Char Select is like that toadette model but thats basically it
